toolchain/sem_ir/file.cpp

```diff
@@ -105,9 +110,11 @@ File::File(SharedValueStores& value_stores, std::string filename,
   static constexpr auto BuiltinIR = CrossRefIRId(0);
   for (auto [i, inst] : llvm::enumerate(builtins->insts_.array_ref())) {
     // We can reuse builtin type IDs because they're special-cased values.
```

Comment:
This line feels like it should be more closely associated with the `inst.type_id()`; it sort of was before, but you've added `builtin_id` on its own line now. Maybe the comment is a little unclear about what it means though?

Reply:
Done.